app/containers/Forms/SummaryEmissionGasFields.tsx

```diff
@@ -59,7 +59,11 @@ const EmissionGasFields: React.FunctionComponent<FieldProps> = (props) => {
           (props.formData.annualEmission || previousEmission) ? (
             <>
               <span className="diffFrom">
-                {previousEmission ? previousEmission : <i>[No Data Entered]</i>}
+                {previousEmission || previousEmission === 0 ? (
+                  previousEmission
+                ) : (
+                  <i>[No Data Entered]</i>
+                )}
```

Comment:
```suggestion
              previousEmission ?? <i>[No Data Entered]</i>
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator